src/overlays/actors/ovl_En_Brob/z_en_brob.c

```diff
@@ -322,7 +325,7 @@ void EnBrob_Draw(Actor* thisx, PlayState* play) {
     EnBrob* this = (EnBrob*)thisx;
 
     Gfx_SetupDL_25Opa(play->state.gfxCtx);
-    Matrix_Translate(0.0f, this->unk_1AE, 0.0f, MTXMODE_APPLY);
+    Matrix_Translate(0.0f, this->yOffset, 0.0f, MTXMODE_APPLY);
     SkelAnime_DrawFlexOpa(play, this->skelAnime.skeleton, this->skelAnime.jointTable, this->skelAnime.dListCount, NULL,
```

Comment:
maybe `modelOffsetY`, to make clear its only the model/skeleton and not the whole actor? this is prob okay though

Reply:
I did this based off a similar translation I noticed in Goma with `eggYOffset`, modified slightly as `brobYOffset` felt redundant. I can change it to `modelOffsetY` as that does align more with what is actually happening.

Reply:
yeah id say specifying model is good to be clear its just an offset of the visuals and nothing else